ibm/service/power/ibm_pi_constants.go

```diff
@@ -84,6 +98,8 @@ const (
 	Arg_StorageType                         = "pi_storage_type"
 	Arg_SysType                             = "pi_sys_type"
 	Arg_TargetStorageTier                   = "pi_target_storage_tier"
+	Arg_TCPFlags                            = "pi_tcp_flags"
```

Comment:
This does not appear to be used anywhere

Reply:
updated